README.md

```diff
@@ -110,7 +110,7 @@ You can run `rustfmt --help` for more information.
 * [Emacs](https://github.com/rust-lang/rust-mode)
 * [Sublime Text 3](https://packagecontrol.io/packages/BeautifyRust)
 * [Atom](atom.md)
-* Visual Studio Code using [RustyCode](https://github.com/saviorisdead/RustyCode) or [vsc-rustfmt](https://github.com/Connorcpu/vsc-rustfmt)
+* Visual Studio Code using [vscode-rust](https://github.com/editor-rs/vscode-rust) or [vsc-rustfmt](https://github.com/Connorcpu/vsc-rustfmt)
```

Comment:
I prefer to link https://github.com/jonathandturner/rls_vscode, either instead of or as well as vscode-rust